vulnfeeds/cpp/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
nit: Is this script only used for building the image manually and do we actually need it?

Reply:
There's no other good breadcrumb trail near the code/Dockerfile for what the actual image name is.

If we can come up with some other convention to help the unfamiliar figure out the name of the Docker image that gets built, I'd be fine with delousing this.

Reply:
Yeah, makes sense - happy to keep it.
In the future, I'd like to think of some other convention - osv.dev having a bunch of unused build scripts can be a bit confusing sometimes.